Sources/NIOHTTP1/NIOTypedHTTPServerUpgradeHandler.swift

```diff
@@ -269,14 +269,19 @@ public final class NIOTypedHTTPServerUpgradeHandler<UpgradeResult: Sendable>: Ch
         request: HTTPRequestHead,
         allHeaderNames: Set<String>,
         connectionHeader: Set<String>
-    ) -> EventLoopFuture<
-        (upgrader: any NIOTypedHTTPServerProtocolUpgrader<UpgradeResult>, responseHeaders: HTTPHeaders, proto: String)?
-    > {
+    )
+        -> EventLoopFuture<
+            (
+                upgrader: any NIOTypedHTTPServerProtocolUpgrader<UpgradeResult>, responseHeaders: HTTPHeaders,
+                proto: String
```

Comment:
Given the weird formatting can we do one param per line?

```suggestion
                upgrader: any NIOTypedHTTPServerProtocolUpgrader<UpgradeResult>, 
                responseHeaders: HTTPHeaders,
                proto: String
```

Reply:
Sorry, there must have been more than one instance or something - I definitely tweaked formatting in code that looked like this https://github.com/apple/swift-nio/pull/3115/commits/4a5073e8655a3a344ddd0e8317f59833e50297d9#diff-df1f0e00e4593ea99b5fed7e3a37f6415e9531d25dcb6994d6ba0f0a8ecf6cebR316